admin-tools.Dockerfile

```
@@ -46,4 +46,4 @@ COPY --from=admin-tools-builder /home/builder/temporal/schema /etc/temporal/sche
 COPY --from=admin-tools-builder /home/builder/temporal/tdbg /usr/local/bin
 
 # Keep the container running.
-ENTRYPOINT ["tail", "-f", "/dev/null"]
+ENTRYPOINT ["tini", "--", "tail", "-f", "/dev/null"]
```

Unrelated to this PR, but while you're here...

```suggestion
ENTRYPOINT ["tini", "--", "sleep", "infinity"]
```

That might be a bit more intuitive that we're trying to deliberately keep the container alive. Feel free to ignore this though, just an aside.

---

TIL.

However, I found this comment on StackOverflow:
```
Regarding this, I made some research I documented in a separate answer. To summarize: infinity is converted in C from "string" to a double. Then that double is truncated to the maximum values allowed timespec, which means a very large amount of seconds (architecture-dependant) but, in theory, finite.
```